src/nvim/optionstr.c

```diff
@@ -1497,35 +1497,12 @@ static void did_set_vartabstop(buf_T *buf, win_T *win, char **varp, char **errms
   }
 }
 
-static void did_set_optexpr(buf_T *buf, win_T *win, char **varp, char **gvarp)
+static void did_set_optexpr(win_T *win, char **p_opt, char **varp, char **gvarp)
```

Comment:
It isn't in case of using `:setglobal` for a buffer-local option. However there seems to be a bug in the code here, so it's better to avoid changing this in this PR and fix it later.